requirements/base.txt

```diff
@@ -40,20 +41,12 @@ django==2.2.24
     #   edx-auth-backends
     #   edx-django-release-util
     #   edx-django-utils
-django-cors-headers==2.2.1
-    # via
-    #   -c requirements/constraints.txt
-    #   -r requirements/base.in
 django-crum==0.7.9
     # via edx-django-utils
-django-environ-2==2.1.0
-    # via
-    #   -c requirements/constraints.txt
-    #   -r requirements/base.in
-django-filter==2.1.0
-    # via
-    #   -c requirements/constraints.txt
-    #   -r requirements/base.in
+django-environ==0.7.0
+    # via -r requirements/base.in
+django-filter==21.1
```

Comment:
Constraint on this has been removed. Though there do not seem to be any backwards incompatible changes since 2.1.0: https://github.com/carltongibson/django-filter/blob/main/CHANGES.rst